src/librustdoc/html/static/rustdoc.css

```diff
@@ -1261,7 +1138,177 @@ h4 > .important-traits {
 	margin-bottom: 6px;
 }
 
+.theme-picker {
+	position: absolute;
```

Comment:
Does this commit just move entries around, or does it actually change any semantics too?

Reply:
The second commit is a cleanup.